rust/kcl-lib/tests/assembly_non_default_units/ops.snap

```diff
@@ -18,6 +18,72 @@ description: Operations executed assembly_non_default_units.kcl
     "type": "StdLibCall",
     "unlabeledArg": null
   },
+  {
+    "type": "UserDefinedFunctionCall",
+    "name": "circle",
```

Comment:
Hmm, Operations treat KCL-defined stdlib functions as user-defined. I believe this is fine for now since sketch functions like `circle` aren't displayed in the Feature Tree. But we'll probably need to fix this eventually, as we flesh it out.

Reply:
Yep, that's why there's that TODO in exec_ast :-)